### PR TITLE
tests: add test for issue 7241/7242 for 7

### PR DESCRIPTION
**Pull request by victorjulien**
Add test that works with Suricata 7.

### CODE REVIEW

**Review by catenacyber — APPROVED** [review]
Ok for me : bug-7241-01-8plus cannot work for 7 even with the suricata backport because `app-layer-protocol:!tls,either` with the `either` is a new feature in 8 ( and also maybe prefilter for app-layer-protocol)

File: tests/bug-7241-02-pre8/test.rules [hunk 1]
```diff
@@ -0,0 +1 @@
+drop tcp any any -> any any (flow:established; app-layer-protocol:!tls; sid:1;)
```
Comment (catenacyber) [inline tests/bug-7241-02-pre8/test.rules:1]: nit : could add a README.md, especially for explaining why we have 2 different tests for 7 and 8

**Comment by catenacyber** [comment]
Oh, I guess we should test prefilter as well cf https://github.com/OISF/suricata/pull/11750/files#r1753510437

**Comment by victorjulien** [comment]
replaced by #2035

**Pull request closed without merging**